docs/source/conf.py

```diff
@@ -14,22 +14,13 @@
 import os
 import shutil
 import sys
-from pathlib import Path
 
 import syne_tune
 
 
 sys.path.insert(0, os.path.abspath("../../"))
 
 
-def copy_notebooks_into_docs_folder(app):
```

Comment:
this was done to have the notebook shown in the doc right? dont mind not having it but then perhaps we may want to remove the notebook too?